src/stories/atoms/Shape/Shape.stories.tsx

```diff
@@ -0,0 +1,44 @@
+// Global Imports
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+// Local Imports
+import { Shape } from 'stories/atoms/Shape/Shape';
+
+export default {
+  title: 'Design System/Atoms/Shape',
+  component: Shape,
+  decorators: [
+    (Story) => (
+      <div
+        style={{
+          backgroundColor: '#C4C4C4',
```

Comment:
Can you refactor to use the theme colors? Check out the `decorators` in other story files.

Reply:
Done in [48da7cf](https://github.com/wijohnst/thriving-park/pull/144/commits/48da7cfe7e5d6efe572ac383263d4923152f71ae)